FDTD/FDTD.f90

```diff
@@ -419,11 +459,25 @@ subroutine dt_evolve_Ac_2dc()
   integer :: ix_m, iy_m
   real(8) :: Y, RR(3) ! rot rot Ac
   ! (written by M.Uemoto on 2016-11-22)
-  Ac_old_m=Ac_m
-  Ac_m=Ac_new_m
+
+!$omp parallel do collapse(2) default(none) &
+!$    private(iy_m,ix_m) &
+!$    shared(NYvacB_m,NYvacT_m,NXvacL_m,NXvacR_m,Ac_m,Ac_old_m,Ac_new_m)
   do iy_m=NYvacB_m,NYvacT_m
-    Y = (iy_m - 0.50d0) * HY_m
     do ix_m=NXvacL_m,NXvacR_m
```

Comment:
We need to extend the range of do loops,  `do ix_m=NXvacL_m-1,NXvacR_m+1` and `do iy_m=NYvacB_m-1,NYvacT_m+1` because the size of these arrays are slightly larger than the present range.